examples/llm_hello_world.py

```diff
@@ -27,6 +29,15 @@ async def main():
     if hasattr(llm, "completion"):
         logger.info(llm.completion(hello_msg))
 
+    # check llm-vision capacity if it supports
+    invoice_path = Path(__file__).parent.joinpath("..", "tests", "data", "invoices", "invoice-2.png")
+    img_base64 = encode_image(invoice_path)
+    try:
```

Comment:
No exception capture is needed.

Reply:
Due to we don't know which llm will be configured by the user which may don't have the image chat ability, so add a try here. Like someone will try glm-4 first.

Reply:
If the LLM doesn't have this capability, then the exception should be thrown to the user